zebra-chain/src/block/height.rs

```diff
@@ -48,6 +48,14 @@ impl Height {
     /// `Height::MAX.0` can't be used in match range patterns, use this
     /// alias instead.
     pub const MAX_AS_U32: u32 = Self::MAX.0;
+
+    /// The maximum expiration Height that is allowed in coinbase transactions
```

Comment:
Confusing Documentation: The consensus rules are:

> [Overwinter to Canopy inclusive, pre-NU5] nExpiryHeight MUST be less than or equal to 499999999.
> 
> [NU5 onward] nExpiryHeight MUST be less than or equal to 499999999 for non-coinbase transactions.

Which means:
```suggestion
    /// The maximum expiration Height that is allowed in all transactions
```